src/Card/index.jsx

```diff
@@ -53,7 +54,14 @@ export { default as CardGroup } from 'react-bootstrap/CardGroup';
 export { default as CardGrid } from './CardGrid';
 
 Card.propTypes = {
-  ...BaseCard.propTypes,
+  /** Specifies a background-color that is displayed on the Card. */
+  bg: PropTypes.oneOf(COLOR_VARIANTS),
```

Comment:
I don't think these props work as expected because we customized react-bootstrap's `Card` too heavily. I'd rather not display them in order not to confuse consumers.